README.md

```diff
@@ -5,3 +5,8 @@ experimenting terraform, RDS and kubernates
 Contributors:
 Navadeep M &
 Surya V
+
+Created new branch for individual work.
+Please proceed on with your work in this branch.
+
+### This is new change
```

Comment:
remove extra line

Reply:
@suryaval i have made necessary changes. Please review the change.